Reqnroll/Formatters/RuntimeSupport/FeatureLevelCucumberMessages.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Io.Cucumber.Messages.Types;
+using System;
+using System.Collections.Generic;
+
+namespace Reqnroll.Formatters.RuntimeSupport;
+
+/// <summary>
+/// This class is used at Code Generation time to provide serialized representations of the Source, GherkinDocument, and Pickles 
+/// to be used at runtime.
+/// </summary>
+public class FeatureLevelCucumberMessages(Func<Source> source, Func<GherkinDocument> gherkinDocument, Func<IEnumerable<Pickle>> pickles, string location)
+{
+    public string Location { get; } = location;
```

Comment:
(3) The `Location` does not seem to be used.

Reply:
Yes, this can be removed. At one time was considering reading the source text at runtime but abandoned that approach; forgot to remove this attribute afterwards.